irondb-relay.md

```diff
@@ -0,0 +1,609 @@
+## IRONdb-relay
+
+The IRONdb-relay, like the carbon-relay or the carbon-c-relay is a metrics data
+router that takes carbon TEXT format metrics and routes them to the appropriate
+IRONdb storage node. 
+
+Since IRONdb uses SHA256 hashing to route metrics to IRONdb nodes, it is
+incompatible with routing options that exist in carbon-c-relay and carbon-relay.
+In addition, it provides advanced aggregation and filtering functions.
+
+Features
+===========
+
+* Ingests TEXT carbon format metrics on a configurable port
+
+  `foo.bar.baz 1234.56 1507724786`
+  
+* Routes to primary owner of the metric name and then subsequent nodes if the
+  primary is down.
+
+* [Aggregation](#modules) of incoming metrics based on regular expressions with support for
+  SUM, AVG, MIN, MAX, p0, p25, p50, p95, p99, p100
+  
+* [Blacklist and whitelist filtering](#modules) of metrics based on regular expressions
+  
+* Durable delivery of metrics using write head logs
+  
+Installation
+================
+
+IRONdb-relay requires one of the following operating systems:
+* RHEL/CentOS, version 7.x.
+* Ubuntu 16.04 LTS.
+
+Additionally, IRONdb-relay requires the [ZFS](http://open-zfs.org/) filesystem.
+This is available natively on Ubuntu, but for EL7 installs, you will
+need to obtain ZFS from
+the [ZFS on Linux](https://github.com/zfsonlinux/zfs/wiki/Getting-Started)
+project. The setup script expects a zpool to exist, but you do not need to
+create any filesystems or directories ahead of time. Please refer to the
+appendix [ZFS Setup Guide](/zfs-guide.md) for details and examples.
+
+The following network protocols and ports are utilized. These are defaults and
+may be changed via configuration files.
+
+* 2003/tcp (Carbon plaintext submission)
+* 8112/tcp (admin UI, HTTP REST API)
```

Comment:
The relay listens on 8112?  Surprising to me, but if true, ignore this comment.

Reply:
yes.  Has an Admin UI